### DISCUSSION TITLE
FFmpegPCMAudio functions open a console window while executing.

### DISCUSSION THREAD

**Discussion by IC0okieI** [Rapptz/discord.py#6373] [Q&A]
Hello,
I have a question about the FFmpegPCMAudio function. I wrote a discord bot which connects after somebody joins a channel to plays some greeting sounds. I want the script to run in the background, so no console window appears when starting the script. I use a .bat and a .vbs file to start it. Everything works fine, no console log appears, but everytime the bot creates an audio source from mp3 files, it prints out a line "_[mp3 @ 0000020168869040] Estimating duration from bitrate, this may be inaccurate_". Therefore a console windows opens for 1sec to display this message and closes afterwards, which is really annoying. I use the FFmpegPCMAudio function to create an audio source from a mp3 file with the "ffmpeg-win64-v4.2.2.exe" executable ("_audio_source = discord.FFmpegPCMAudio(executable="E:/PythonPrograms/DiscordBotHallo/Dateien/ffmpeg-win64-v4.2.2.exe", source=source)_"). I also asked this question about the console log in their Github "https://github.com/imageio/imageio-binaries/issues/14" , but they said it has something to do with the function, not the executable, so i'm asking my question here now. Does somebody know a way to pretend a console to appear when calling this function?
Thanks
Cookie

my full code if needed:
[main.txt](https://github.com/Rapptz/discord.py/files/5911276/main.txt)

**Comment by Rapptz** [comment]
This was reported already at https://github.com/Rapptz/discord.py/issues/6038 and fixed in v1.6. Make sure your library is up to date by using `py -3 -m pip install -U discord.py[voice]`.
